raml-parser-2/src/test/java/org/raml/v2/api/TypesTest.java

```diff
@@ -0,0 +1,56 @@
+package org.raml.v2.api;
+
+import org.junit.Test;
+import org.raml.v2.api.model.common.ValidationResult;
+import org.raml.v2.api.model.v10.datamodel.TypeDeclaration;
+
+import java.io.File;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Jean-Philippe Belanger on 9/24/16.
+ * Just potential zeroes and ones
+ */
+public class TypesTest {
```

Comment:
Instead of writing a new test class you can leverage ApiModelParserTestCase and just provide the `input.raml` and `model.json` output